x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/connector_configuration.tsx

```diff
@@ -270,6 +270,30 @@ export const ConnectorConfiguration: React.FC = () => {
                 {
                   children: (
                     <EuiFlexGroup direction="column">
+                      <EuiFlexItem>
+                        <EuiCallOut
```

Comment:
Should we render this warning conditionally only when no index is attached?

Reply:
Oops, good catch, just forgot to add that. Done in latest :)